spec/models/data_request_spec.rb

```diff
@@ -133,17 +133,6 @@
     end
   end
 
-  describe '#case' do
-    subject(:data_request) { build :data_request }
-
-    it { should be_valid }
-
-    it 'is restricted to Offender SAR at present' do
```

Comment:
Might be worth thinking about this in an additive way once we have the tests passing i.e. "It is restricted to Offender SAR and Offender Sar Complaints at present"